### DISCUSSION TITLE
Is it possible to rename a Prisma 1 schema in Postgres?

### DISCUSSION THREAD

**Discussion by rootedsoftware** [prisma/prisma#4430] [Q&A] [answered]
I am doing a backup and restore of my DB to a staging environment. I'd like to rename my postgres schema from app$prod to app$stage. I have found that this is possible using the 
```
ALTER SCHEMA app$prod
RENAME TO app$stage;
```
commands, but after a rename I get errors in all my queries. 

I've also tried to make the change in my prisma.yml file and then doing a ```prisma deploy```, but that doesn't create a new schema, it just breaks the deployment.

**Comment by ryands17** [comment, marked as answer]
Hey @c316 👋 
This is the discussions section for Prisma 2. It would be great if you could add your query [here](https://github.com/prisma/prisma1/issues) :)

**Reply by rootedsoftware** [reply]
Thanks, I did that.

https://github.com/prisma/prisma1/issues/5133

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
